app.js

```diff
@@ -240,14 +240,13 @@ d.run(() => {
 							}
```

Comment:
Should we not remove the peers which dns lookup fail?

`return callback(err);` should do that.

Reply:
Github didn't allow me to select the correct line but the line I'm referring to is app.js:239

Reply:
[async.reflect](https://caolan.github.io/async/reflect.js.html) collects all the task execution results into one single object `results` passed through a the callback on line 246 with either an error attribute or a value property. So the `results` object contains all the task execution results and then it's filter and mapped so the peers list only contains resolved dns' IPs

Reply:
Nice! I thought it would have both `error` and `value` in this case. All good then.